From ef91d7c2b7f2cf591d0746317e6588ecc85281ca Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.uw@renesas.com>
Date: Mon, 1 Oct 2018 15:56:23 +0700
Subject: [PATCH 325/504] ARM: dts: iwg21m: Change to use vin1_clk_b pin group

The iWave RZ/G1H board (iwg21m) use vin1_clk_b pin group instead of
vin1_clk. The current pin setting (vin1_clk) made conflict with avb.

Signed-off-by: Binh Nguyen <binh.nguyen.uw@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7742-iwg21m.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a7742-iwg21m.dts b/arch/arm/boot/dts/r8a7742-iwg21m.dts
index 438b9b5..355eb1c 100644
--- a/arch/arm/boot/dts/r8a7742-iwg21m.dts
+++ b/arch/arm/boot/dts/r8a7742-iwg21m.dts
@@ -461,7 +461,7 @@
 	};
 
 	vin1_pins: vin1 {
-		groups = "vin1_data8", "vin1_clk";
+		groups = "vin1_data8", "vin1_clk_b";
 		function = "vin1";
 	};
 
-- 
2.7.4

